SS2-Project/Assets/Starstorm2/Modules/SS2EventCard.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using UnityEngine;
+using Moonstorm.AddressableAssets;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using RoR2;
+using R2API;
+using UnityEngine.AddressableAssets;
+
+namespace Moonstorm.Starstorm2
+{
+    public class SS2EventCard : EventCard
+    {
+        [Serializable]
+        public class EventVFX
+        {
+            public R2API.DirectorAPI.Stage stageEnum;
+            public string customStageName;
+            public GameObject effectPrefab;
+
+            public GameObject EffectPrefab { get; internal set; }
```

Comment:
Pointless Property? looks like the create dictionary uses this instead of the field

Reply:
oh I think it got autogenerated bc typo. Yeah lemme fix